.ci/appveyor.yml

```diff
@@ -29,7 +29,7 @@ install:
   # Prepend newly installed Python to the PATH of this build (this cannot be
   # done from inside the powershell script as it would require to restart
   # the parent CMD process).
-  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "SET PATH=C:\\tools\\xmllint\\bin;%PYTHON%;%PYTHON%\\Scripts;%PATH%"
```

Comment:
This addition should be on the next line.
The reason is that this line is identical to https://github.com/ogrisel/python-appveyor-demo/blob/f54ec35/appveyor.yml#L103